templates/docs/examples/patterns/tabs/content-dark.html

```diff
@@ -0,0 +1,58 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Tabs / Tabbed Content{% endblock %}
+
+{% block standalone_css %}patterns_tabs{% endblock %}
+
+{% block content %}
+<div class="p-strip" style="background: #262626;">
```

Comment:
Why not just `p-strip--dark` without hardcoded background?

Reply:
fixed